src/main/java/org/testng/reporters/EmailableReporter.java

```diff
@@ -50,6 +50,18 @@
 
   private int m_methodIndex;
 
+  private String fileName = "emailable-report.html";
```

Comment:
There's a setter via which this field can be altered. So this cannot be changed into a `final` variable.